src/styles/Panel.css

```diff
@@ -38,16 +38,136 @@
 .panel-image {
   border-radius: 500px;
   border: 1px solid #dddcdf;
-  width: 205px;
+  width: 200px;
```

Comment:
i liked the slightly oblong image, but if you feel the circular looks better, I don't mind

Reply:
ah ok i switched it back to 205px